### DISCUSSION TITLE
How to pass sentence or multiple words to multiple arguments?

### DISCUSSION THREAD

**Discussion by DennisMinn** [TwitchIO/TwitchIO#449] [Questions] [answered]
I want to pass multiple arguments enclosed in either ' or ", but on the first argument is properly formatted, while the second argument is cut off. Is there a way to modify how the arguments is parsed?

```
class Bot(commands.Bot):
    def __init__(self):
        super().__init__(token='<access token>', prefix='!', initial_channels = ['<initial channel>'])

    @commands.command(name='addQA')
    async def add_qa(self, context: commands.Context, question: str, answer: str):
        print(question, answer)
```
Input from `<initial channel>` chatroom
```
!addQA "this is a question" "this is the answer"
```

Current Output
```
this is a question "this
```

Desired Output
```
this is a question this is the answer
```

**Comment by chillymosh** [comment, marked as answer]
@DennisMinn It does look like there is an issue with the tokeniser in regards to this behaviour. Are you able to test what I posted in
https://github.com/PythonistaGuild/TwitchIO/issues/450

**Reply by DennisMinn** [reply]
Works as intended, thank you!
